tests/end_to_end/test_brute_force/data/playbooks/credentials.yaml

```diff
@@ -0,0 +1,18 @@
+---
+    - name: Get credentials
+      hosts: wazuh-manager
+      tasks:
+
+        - name: Unzip wazuh install files
+          unarchive:
+            src: /home/vagrant/wazuh-install-files.tar
+            dest: /home/vagrant
+            remote_src: yes
+          become: True
+
+        - name: Get passwords file
+          fetch:
+            src: /home/vagrant/wazuh-install-files/passwords.wazuh
+            dest: /tmp/
+            flat: yes
+          become: True
```

Comment:
```suggestion
- name: Get credentials
  hosts: wazuh-manager
  tasks:

    - name: Unzip wazuh install files
      unarchive:
        src: /home/vagrant/wazuh-install-files.tar
        dest: /home/vagrant
        remote_src: yes
      become: True

    - name: Get passwords file
      fetch:
        src: /home/vagrant/wazuh-install-files/passwords.wazuh
        dest: /tmp/
        flat: yes
      become: True
```

Reply:
Don't tabulate the beginning. Check out the rest of the playbooks. Apply this to all files.

Reply:
Done